keyboards/darkproject/kd87a_bfg_edition/kd87a_bfg_edition.c

```diff
@@ -0,0 +1,294 @@
+    { 44,    0}, //   3, k30, F3
+    { 55,    0}, //   4, k40, F4
+    { 66,    0}, //   5, k50, F5
+    { 77,    0}, //   6, k60, F6
+    { 88,    0}, //   7, k70, F7
+    { 99,    0}, //   8, k80, F8
+    { 110,   0}, //   9, k90, F9
+    { 121,   0}, //  10, ka0, F10
+    { 132,   0}, //  11, kb0, F11
+    { 143,   0}, //  12, kc0, F12
+    { 154,   0}, //  13, kd0, Printscreen
+    { 165,   0}, //  14, k56, SL
+    { 175,   0}, //  15, k56, PB
+
+   
+    {  11,  11}, //  16, k01, `
+    {  22,  11}, //  17, k11, 1
+    {  33,  11}, //  18, k21, 2
+    {  44,  11}, //  19, k31, 3
+    {  55,  11}, //  20, k41, 4
+    {  66,  11}, //  21, k51, 5
+    {  77,  11}, //  22, k61, 6
+    {  88,  11}, //  23, k71, 7
+    {  99,  11}, //  24, k81, 8
+    { 110,  11}, //  25, k91, 9
+    { 121,  11}, //  26, ka1, 0
+    { 132,  11}, //  27, kb1, -
+    { 143,  11}, //  28, kc1, =
+    { 154,  11}, //  29, kd1, Backspace
+    { 165,  11}, //  30, k16, Ins
+    { 175,  11}, //  31, k16, Home
+    { 185,  11}, //  32, k16, PGUP
+  
+    {  11,  22}, //  30, k02, Tab
+    {  22,  22}, //  31, k12, Q
+    {  33,  22}, //  32, k22, W
+    {  44,  22}, //  33, k32, E
+    {  55,  22}, //  34, k42, R
+    {  66,  22}, //  35, k52, T
+    {  77,  22}, //  36, k62, Y
+    {  88,  22}, //  37, k72, U
+    {  99,  22}, //  38, k82, I
+    { 110,  22}, //  39, k92, O
+    { 121,  22}, //  40, ka2, P
+    { 132,  22}, //  41, kb2, [
+    { 143,  22}, //  42, kc2, ]
+    { 154,  22}, //  43, kd3, "\\"
+    { 165,  22}, //  44, k06, Del
+    { 175,  22}, //  29, k16, END
+    { 185,  22}, //  29, k16, PGDN
+
+    {  11,  33}, //  45, k03, Caps Lock
+    {  22,  33}, //  46, k13, A
+    {  33,  33}, //  47, k23, S
+    {  44,  33}, //  48, k33, D
+    {  55,  33}, //  49, k43, F
+    {  66,  33}, //  50, k53, G
+    {  77,  33}, //  51, k63, H
+    {  88,  33}, //  52, k73, J
+    {  99,  33}, //  53, k83, K
+    { 110,  33}, //  54, k93, L
+    { 121,  33}, //  55, ka3, ;
+    { 132,  33}, //  56, kb3, '
+    { 154,  33}, //  57, kc3, Enter
+
+   
+    {  11,  44}, //  59, k04, Shift_L
+    {  33,  44}, //  60, k24, Z
+    {  44,  44}, //  61, k34, X
+    {  55,  44}, //  62, k44, C
+    {  66,  44}, //  63, k54, V
+    {  77,  44}, //  64, k64, B
+    {  88,  44}, //  65, k74, N
+    {  99,  44}, //  66, k84, M
+    { 110,  44}, //  67, k94, ,
+    { 121,  44}, //  68, ka4, .
+    { 132,  44}, //  69, kb4, /
+    { 143,  44}, //  70, kd4, Shift_R
+    { 154,  44}, //  70, k17, Up
+
+
+    {  11,  55}, //  73, k05, Ctrl_L
+    {  22,  55}, //  74, k15, Win_L
+    {  33,  55}, //  75, k25, Alt_L
+    {  77,  55}, //  76, k65, Space
+    { 110,  55}, //  77, k95, Alt_R
+    { 121,  55}, //  78, ka5, FN
+    { 132,  55}, //  29, k16, App
+    { 143,  55}, //  79, kc5, Ctrl_R
+    { 154,  55}, //  80, k07, Left
+    { 165,  55}, //  81, k27, Down
+    { 176,  55}, //  82, k37, Right
+}, 
+{
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
```

Comment:
All alpha keys, no mod keys, except the one? That appears to be caps lock?

Reply:
My point here is that you do actually have modifier keys, but none of the keys are marked as such.  And the caps lock key should use 0x8, as that is the flag for indicators lights. 

https://docs.qmk.fm/#/feature_rgb_matrix?id=flags